### PR TITLE
Added chujia

### PR DESCRIPTION
**Pull request by ChujiaGuo**
@ahsu1230

### CODE REVIEW

**Comment by ahsu1230** [comment]
Looks like there's a merge conflict because other people are changing the same thing at the same time. Look here to help you solve it, starting from step 4. https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

File: css/style.css [hunk 1]
```diff
@@ -41,9 +41,14 @@ h3 {
...
+
+.chujia{
+  color: #30FFEA
+
 .serena2{
   color: #B47EDE;
 }
 .daniel{
   color: #FF33FF;
+
```
Comment (ahsu1230) [inline css/style.css:54]: remove extra space

File: css/style.css [hunk 2]
```diff
@@ -41,9 +41,14 @@ h3 {
 .fred {
   color: #3471EB;
 }
+
+.chujia{
+  color: #30FFEA
+
```
Comment (ahsu1230) [inline css/style.css:48]: you're missing a `}`

File: index.html [hunk 3]
```diff
@@ -19,8 +19,13 @@ <h1>Leave your footprint here!</h1>
 			<li><h3 class="ariel">Ariel was here!</h3></li>
 			<li><h3 class="ariel2">Ariel was here, again!</h3></li>
 			<li><h3 class="fred">Frederick was here!</h3></li>
+
```
Comment (ahsu1230) [inline index.html]: remove extra spaces here, line 25 and line 28

File: index.html [hunk 4]
```diff
@@ -19,8 +19,13 @@ <h1>Leave your footprint here!</h1>
...
+			<li><h3 class="serena">Serena was here!</h3></li>
```
Comment (ahsu1230) [inline index.html]: serena actually put this one in by accident. Remove this one and keep the one on line 26 (serena2)

**Review by ahsu1230 — CHANGES_REQUESTED** [review]
I added a few comments. Please address those simple changes and you're good to go!

**Comment by ahsu1230** [comment]
If you need additional help, @austinhsu14 can help. Contact him via slack.

File: index.html (hunk 4 above)
Comment (ChujiaGuo) [reply index.html]: Alright, those should be fixed now.

**Review by ahsu1230 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**